### PR TITLE
CNV-38594: RN Known issue PVC data upload form does not work

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.16

Issue: [CNV-38594](https://issues.redhat.com//browse/CNV-38594)

Link to docs preview: https://78390--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-16-release-notes.html#virt-4-16-known-issues

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by max-cx** [comment]
@sjhala-ccs, :slightly_smiling_face: I am in training for the peer-review squad. **LGTM**, and here are a few global comments about this file that are good to know about and of course outside of the scope of this PR:
- You might want to include a sentence or short introductory paragraph after the H1 heading that would serve as the abstract for the file.
- For "OpenShift Data Foundation (ODF)", you might want to consider using the existing attribute in https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc#L233
- Please see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-subsections-level-1-and-lower-headings to consider looking into whether heading anchors in this file use the correct syntax: `[id="module-name_{context}"]`.

@dfitzmau, could you please have a final look at this PR and my suggestions above?

**Comment by dfitzmau** [comment]
Hi @sjhala-ccs . I'm curius to Max's last point, why are `context` variables dropped from ID anchors in this file?

The [core OCP RN](https://raw.githubusercontent.com/openshift/openshift-docs/enterprise-4.16/release_notes/ocp-4-16-release-notes.adoc) doc uses the variable in each ID anchor.

File: virt/release_notes/virt-4-16-release-notes.adoc [hunk 1]
```diff
@@ -231,4 +231,7 @@ Legacy OpenSSL clients that do not support EMS or TLS 1.3 now cannot connect to
 ==== Web console
 //CNV-38293: 4.16 - Ronen Sde-Or said this was still an issue
 * When you first deploy an {product-title} cluster, creating VMs from templates or instance types by using the web console, fails if you do not have `cluster-admin` permissions.
```
Comment (dfitzmau) [inline virt/release_notes/virt-4-16-release-notes.adoc:233]:
```suggestion
* When you deployed an {product-title} cluster, and then created VMs from templates or instance types by using the web console, the VMs failed if you did not have `cluster-admin` permissions. As a workaround, the cluster administrator must first xref:../../nodes/pods/nodes-pods-configmaps.adoc#nodes-pods-configmap-create-from-console_configmaps[create a config map] to provide users access to use templates and instance types to create VMs. (link:https://issues.redhat.com/browse/CNV-38284[*CNV-38284*])
```

File: virt/release_notes/virt-4-16-release-notes.adoc [hunk 2]
```diff
@@ -231,4 +231,7 @@ Legacy OpenSSL clients that do not support EMS or TLS 1.3 now cannot connect to
...
-** As a workaround, the cluster administrator must first xref:../../nodes/pods/nodes-pods-configmaps.adoc#nodes-pods-configmap-create-from-console_configmaps[create a config map] to enable other users to use templates and instance types to create VMs. (link: https://issues.redhat.com/browse/CNV-38284[*CNV-38284*])
\ No newline at end of file
+** As a workaround, the cluster administrator must first xref:../../nodes/pods/nodes-pods-configmaps.adoc#nodes-pods-configmap-create-from-console_configmaps[create a config map] to enable other users to use templates and instance types to create VMs. (link:https://issues.redhat.com/browse/CNV-38284[*CNV-38284*])
+
+//CNV-38594: 4.16 known issue
+* When creating a persistent volume claim (PVC) in the web console by selecting *With Data upload form*, uploading data to the PVC by using the *Upload Data* field fails. (link:https://issues.redhat.com/browse/CNV-37607[*CNV-37607*])
```
Comment (dfitzmau) [inline virt/release_notes/virt-4-16-release-notes.adoc:237]:
```suggestion
* When you created a persistent volume claim (PVC) in the web console by clicking/selecting the *With Data upload form* button/option, uploading data to the PVC by using the *Upload Data* field fails. (link:https://issues.redhat.com/browse/CNV-37607[*CNV-37607*])
```

**Review by dfitzmau — COMMENTED** [review]
Added minor suggestions and one curiosity question. Please see Max's comments also.

File: virt/release_notes/virt-4-16-release-notes.adoc (hunk 1 above)
Comment (sjhala-ccs) [reply virt/release_notes/virt-4-16-release-notes.adoc:233]: This is a 4.15 release note that has been carried over. I only fixed the link syntax.

File: virt/release_notes/virt-4-16-release-notes.adoc (hunk 2 above)
Comment (sjhala-ccs) [reply virt/release_notes/virt-4-16-release-notes.adoc:237]: The *With Data upload form* is an item in a drop-down list, so I went with "select" per our [style guidelines](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#references-to-interface-elements).

Perhaps I should change this to:
```suggestion
* When you select *With Data upload form* from the *Create PersistentVolumeClaim* list in the web console, uploading data to the PVC by using the *Upload Data* field fails. (link:https://issues.redhat.com/browse/CNV-37607[*CNV-37607*])
```

**Comment by sjhala-ccs** [comment]
> Hi @sjhala-ccs . I'm curius to Max's last point, why are `context` variables dropped from ID anchors in this file?
> 
> The [core OCP RN](https://raw.githubusercontent.com/openshift/openshift-docs/enterprise-4.16/release_notes/ocp-4-16-release-notes.adoc) doc uses the variable in each ID anchor.

IIRC, adding a  _{context} variable to the end of anchor IDs in an assembly was not required when the guidelines were first put into place, so the headings in the CNV release notes assembly never had the context variables. It probably didn't come up in peer review as well until now. I will pass on the comment to the release notes coordinator because changing all the IDs is out of the scope of this PR.

**Comment by dfitzmau** [comment]
Thanks, @sjhala-ccs . LGTM

/remove-label peer-review-in-progress

/label peer-review-done

**Pull request merged**